### PR TITLE
Update OSCAR information in EC3 docs

### PR DESCRIPTION
**Pull request by micafer**
# Summary

Update OSCAR information in EC3 docs

### CODE REVIEW

File: content/en/users/cloud-compute/ec3/apps/_index.md [hunk 1]
```diff
@@ -41,16 +41,30 @@ please select Mesos + Marathon + Chronos from the list of available LRMS.
 To deploy
 [Serverless computing for data-processing applications](https://www.egi.eu/about/newsletters/serverless-computing-for-data-processing-applications-in-egi/)
 in EGI, please select OSCAR from the list of LRMS (Local Resource Management
-System). Use cases of applications that use the OSCAR framework for
-event-driven high-throughput processing of files:
+System). OSCAR supports data-driven serverless computing for file-processing applications. 
+Services will be triggered in response to a file upload to an object storage backend in order 
+to execute a user-defined shell script inside a container provisioned out of an user-defined 
+Docker image. These will be orchestrated as a Kubernetes batch jobs. The output data will be uploaded
```
Comment (gwarf) [inline content/en/users/cloud-compute/ec3/apps/_index.md:47]: This sentence seems a bit too convoluted/complex, maybe we could try to simplify in a way similar the the proposition below (I may have misunderstood something, so please adapt):
```suggestion
A file upload, to an object storage backend, will trigger the execution of a chosen shell script running inside a user-defined container. These will be orchestrated as a Kubernetes batch jobs. The output data will be uploaded 
```

File: content/en/users/cloud-compute/ec3/apps/_index.md [hunk 2]
```diff
@@ -33,26 +33,44 @@ Check the
...
 [Serverless computing for data-processing applications](https://www.egi.eu/about/newsletters/serverless-computing-for-data-processing-applications-in-egi/)
 in EGI, please select OSCAR from the list of LRMS (Local Resource Management
-System). Use cases of applications that use the OSCAR framework for
-event-driven high-throughput processing of files:
-
-* [Plants Classification](https://github.com/indigo-dc/plant-classification-theano),
+System). OSCAR supports data-driven serverless computing for file-processing
+applications. A file upload, to an object storage backend, will trigger the
+execution of a chosen shell script running inside a user-defined container.
+These will be orchestrated as a Kubernetes batch jobs. The output data will be
```
Comment (enolfc) [inline content/en/users/cloud-compute/ec3/apps/_index.md:47]:
```suggestion
These will be orchestrated as Kubernetes batch jobs. The output data will be
```

File: content/en/users/cloud-compute/ec3/apps/_index.md [hunk 3]
```diff
@@ -33,26 +33,44 @@ Check the
...
+uploaded to any object storage backends support. Synchronous invocations
```
Comment (enolfc) [inline content/en/users/cloud-compute/ec3/apps/_index.md:48]:
```suggestion
uploaded to any object storage that the backends support. Synchronous invocations are
```

Still it's a bit fuzzy how the backend relates to the object storage, probably this is detailed in the OSCAR docs, but here it's confusing. Maybe this whole sentence should be simplified like (just a suggestion, please improve as needed) and the details can be then read at OSCAR's docs

```
File uploads to a supported object storage will trigger the execution of actions inside user-defined
containers that are orchestrated as Kubernetes batch jobs. Output of those actions can be uploaded 
to object storage as well.
```

**Review by enolfc — CHANGES_REQUESTED** [review]
Hi @micafer ! Some suggestions for change

File: content/en/users/cloud-compute/ec3/apps/_index.md (hunk 3 above)
Comment (micafer) [reply content/en/users/cloud-compute/ec3/apps/_index.md]: I have updated the description. Please check if it is clearer now.

File: content/en/users/cloud-compute/ec3/apps/_index.md [hunk 4]
```diff
@@ -33,26 +33,54 @@ Check the
...
 To deploy
 [Serverless computing for data-processing applications](https://www.egi.eu/about/newsletters/serverless-computing-for-data-processing-applications-in-egi/)
 in EGI, please select OSCAR from the list of LRMS (Local Resource Management
-System). Use cases of applications that use the OSCAR framework for
-event-driven high-throughput processing of files:
-
-* [Plants Classification](https://github.com/indigo-dc/plant-classification-theano),
+System). OSCAR supports data-driven serverless computing for file-processing
+applications. A file upload, to the object storage backend [MinIO](http://minio.io),
+will trigger the execution of a chosen shell script running inside a user-defined
```
Comment (enolfc) [inline content/en/users/cloud-compute/ec3/apps/_index.md:46]: maybe just a nitpick, only shell scripts can be run?

File: content/en/users/cloud-compute/ec3/apps/_index.md [hunk 5]
```diff
@@ -33,26 +33,54 @@ Check the
...
+
+As external object storage providers, the following services can be used:
+
+- External [MinIO](https://min.io) servers, which may be in clusters other than the 
+  platform.
+- [Amazon S3](https://aws.amazon.com/s3/), the Amazon's  object storage service that
+  offers industry-leading scalability, data availability, security, and performance
+  in the public Cloud.
+- [Onedata](https://onedata.org/), the global data access solution for science used
+  in the [EGI Federated Cloud](https://datahub.egi.eu/).
```
Comment (enolfc) [inline content/en/users/cloud-compute/ec3/apps/_index.md:59]:
```suggestion
  in the [EGI DataHub](https://datahub.egi.eu/).
```

**Review by enolfc — CHANGES_REQUESTED** [review]
Looking better! Just a very small change requested.

File: content/en/users/cloud-compute/ec3/apps/_index.md (hunk 4 above)
Comment (micafer) [reply content/en/users/cloud-compute/ec3/apps/_index.md:46]: Yes, it must be a shell script (than can execute anything that is inside the container image)

**Review by enolfc — APPROVED** [review]
Good to go!

File: content/en/users/cloud-compute/ec3/apps/_index.md [hunk 6]
```diff
@@ -33,26 +33,54 @@ Check the
...
+System). OSCAR supports data-driven serverless computing for file-processing
+applications. A file upload, to the object storage backend [MinIO](http://minio.io),
+will trigger the execution of a chosen shell script running inside a user-defined
+container. These will be orchestrated as Kubernetes batch jobs. The output data
+will be uploaded to any object storage backends support. Synchronous invocations
+are also available.
+
+As external object storage providers, the following services can be used:
+
+- External [MinIO](https://min.io) servers, which may be in clusters other than the
```
Comment (enolfc) [inline content/en/users/cloud-compute/ec3/apps/_index.md:53]:
```suggestion
- External [MinIO](https://min.io) servers, which may be in clusters other than the
```

**Pull request merged**